### PR TITLE
Add image example

### PR DESCRIPTION
**Pull request by kkristof**

### CODE REVIEW

File: examples/img/img.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+surface = new createXSurface(800, 600);
+gepard = new Gepard(surface);
+
+image = loadPng("dice.png");
+
+gepard.setFillColor(255, 255, 255, 1.0);
+gepard.fillRect(0, 0, surface.width, surface.height);
+
+gepard.drawImage(image, 0, 0, surface.width, surface.height);
+while(!surface.hasToQuit()) {}
```
Comment (szledan) [inline examples/img/img.js]: Missing space: `while (!surface...`

**Review by szledan — CHANGES_REQUESTED** [review]

File: examples/img/img.js (hunk 1 above)
Comment (kkristof) [reply examples/img/img.js]: Thanks, I've updated the branch

**Review by szledan — APPROVED** [review]
LGTMQI +1

**Review by bzsolt — APPROVED** [review]
LGTM

**Pull request merged**